src/spesh/inline.c

```diff
@@ -596,7 +596,7 @@ static MVMSpeshBB * merge_graph(MVMThreadContext *tc, MVMSpeshGraph *inliner,
             MVMSpeshDeoptUseEntry *due = inliner->facts[i][j].usage.deopt_users;
             while (due) {
                 if ((MVMuint32)due->deopt_idx == impl_deopt_idx) {
-                    MVMSpeshOperand o;
+                    MVMSpeshOperand o = {0};
                     o.reg.orig = i;
                     o.reg.i = j;
```

Comment:
Seems ok; both fields of the `reg` variant are set.

The `MVMSpeshOperand` type could use some doc strings, fwiw.